redist/org.goldendict.desktop.appdata.xml

```diff
@@ -34,6 +34,6 @@
   </screenshots>
   <url type="homepage">http://goldendict.org</url>
   <update_contact>https://github.com/goldendict/goldendict</update_contact>
-  <launchable type="desktop-id">goldendict.desktop</launchable>
+  <launchable type="desktop-id">org.goldendict.desktop.desktop</launchable>
```

Comment:
why you didn't update the actual id?

Reply:
never mind me, it's already the case.

Reply:
I'll update it, since we've agreed on this